docs/ingress.md

```diff
@@ -33,16 +47,44 @@ Using a domain that you manage, create a DNS wildcard `A Record` pointing to the
 
 **NOTE:** you are welcome to use `*.draft.example.com` or any other wildcard domain.
 
-Remember the domain you use, it will be needed in the next step of installation as the `basedomain` passed to `draft init`.
+Remember the domain you use, it wiLl be needed in the next step of installation as the `basedomain` passed to `draft init`.
 
 | Name          | Type | Data                      |
 |---------------|------|---------------------------|
 | *.example.com | A    | `<ip address from above>` |
 
+
+### I don't manage a domain
+
+If you don't manage a domain, when you will perform the request, you can use the
+host header to use ingress the host base routing.
+
+```
+$ curl --header Host:<application domain> <ip address from above>
```

Comment:
No it is just like `curl 192.168.99.100` but with the header flag added.
It can either be `curl 192.168.99.100 --header Host:my.domain.com` or `curl --header Host:my.domain.com 192.168.99.100`